app-backend/batch/src/main/scala/com/azavea/rf/batch/stac/ReadStacFeature.scala

```diff
@@ -227,6 +227,14 @@ object ReadStacFeature extends Config with LazyLogging {
     Projected(poly, 3857)
   }
 
+  // PNGs for stac features must be referenced at radiant-nasa-iserv.s3.amazonaws.com/... instead of
+  // at s3.amazonaws.com/bucket/...
+  private def createThumbnailUrl(thumbnailPath: URI, rootUri: URI): URI = {
```

Comment:
This is causing an issue -- not quite sure what's wrong, but I tried http://radiant-nasa-iserv.s3-us-west-2.amazonaws.com/index.html?prefix=2014/04/07/IPR201404072120201137S07593W/ and the thumbnail that I got was 404 https://radiant-nasa-iserv.s3-us-west-2.amazonaws.com.s3.amazonaws.com/2014/04/07/IPR201404072120201137S07593W/IPR201404072120201137S07593W.png

Reply:
Added handling for http paths